db/migrations/20230627112123_create_table_user_addresses.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE user_addresses (
+	id INT NOT NULL AUTO_INCREMENT,
+	user_id INT NOT NULL,
+	nickname VARCHAR(50) NULL,
+	address TEXT NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
- 들여씌기 확인해 주세요.
- FOREIGN KEY 빠진 부분 있습니다. 추가해 주세요.
```suggestion
CREATE TABLE user_addresses (
  id INT NOT NULL AUTO_INCREMENT,
  user_id INT NOT NULL,
  nickname VARCHAR(50) NULL,
  address TEXT NOT NULL,
  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```